README.md

```diff
@@ -1,8 +1,14 @@
 # Waggle Dance
 ![Bee waggle-dancing on a hive.](logo.png "Federating Hive Meta Stores.")
 
+## Overview
+Waggle Dance is a request routing Hive metastore proxy that allows tables to be concurrently accessed across multiple Hive clusters. It was created to tackle the appearance of the dataset silos that arose as our large organization gradually migrated from monolithic on-premises clusters, to cloud based platforms.
```

Comment:
You changed clusters to deployments below, want to change this one too for consistency?